tests/test_exclusion.txt

```diff
@@ -1,10 +1,7 @@
 01-atoti-academy/storytelling-with-atoti/01-Storytelling.ipynb
 01-atoti-academy/storytelling-with-atoti/03-Storytelling.ipynb
-03-use-cases/01-finance/front-office/cumulative-pnl
-03-use-cases/01-finance/front-office/pnl-explained
```

Comment:
Removing notebooks from test exclusion that were successfully upgraded to `0.9.1`.